packages/suite/src/constants/suite/ethStaking.ts

```diff
@@ -1,6 +1,7 @@
 import { BigNumber } from '@trezor/utils/src/bigNumber';
 
 export const MIN_ETH_AMOUNT_FOR_STAKING = new BigNumber(0.1);
+export const MAX_ETH_AMOUNT_FOR_STAKING = new BigNumber(1000000); // 1M
```

Comment:
1_000_000 and then no comment is needed

Reply:
you're right it looks better https://github.com/trezor/trezor-suite/pull/14713/commits/03dbbc780d1e2c8bcce13f33e84bcfa328df7e64